pkg/virt-config/virt-config.go

```diff
@@ -89,7 +89,7 @@ const (
 	DefaultVirtWebhookClientBurst         = 400
 
 	DefaultMaxHotplugRatio   = 4
-	DefaultVMRolloutStrategy = v1.VMRolloutStrategyStage
+	DefaultVMRolloutStrategy = v1.VMRolloutStrategyLiveUpdate
```

Comment:
@enp0s3 I believe this is an unintended change

/hold

Reply:
@fabiand Hey. This is in purpose, I will remove it after testing.